### PR TITLE
adding spec and temp release script

### PR DESCRIPTION
**Pull request by lzap**
This is SCL enabled RHEL SPEC file and a temporary release script until we
sort out how to use tito with repositories with git submodules.

Here is a build: http://koji.katello.org/koji/taskinfo?taskID=36605

### CODE REVIEW

File: foreman-installer.spec [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    %global scl_prefix %{scl}-
+    %global scl_ruby /usr/bin/ruby193-ruby
+%else
+    %global scl_ruby /usr/bin/ruby
+%endif
+
+%global alphatag RC1
+
+Name:       foreman-installer
+Version:    1.2.0
```
Comment (domcleal) [inline foreman-installer.spec:15]: This should be 1.2.9999 in develop.

**Comment by domcleal** [comment]
Please target develop, not master.

File: foreman-installer.spec [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+BuildArch:  noarch
+
+Requires:   %{?scl_prefix}ruby(abi)
+Requires:   %{?scl_prefix}puppet >= 0.24.4
+%if %{?skip_generator:0}%{!?skip_generator:1}
+Requires:   %{?scl_prefix}rubygem-highline
+%endif
+
+%description
+Script to generate answers for puppet-based installer of The Foreman life-cycle
```
Comment (domcleal) [inline foreman-installer.spec:36]: More than just a script, it's the installer too.

**Comment by domcleal** [comment]
I just noticed a bug in our current foreman-installer package which affects this.  The package has epoch set to 1 due to a bug in fpm (https://github.com/jordansissel/fpm/issues/381), so we need to make sure it's set to 1 in this spec file too.

**Comment by lzap** [comment]
aah sorry about PR into master - will close this PR and make a new one.

1.2.9999 - ok but with tito we will probably change this as tito modifies version number.

description fixed yeah - I just wanted not to have summary same as description because then rpmlink throws a warning :-)

epoch - oh right, the only way is to add Epoch: 1 then. adding this line

**Pull request closed without merging**